velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -381,7 +382,7 @@ class NonNumericMinMaxAggregateBase : public exec::Aggregate {
       }
       auto accumulator = value<SingleValueAccumulator>(groups[i]);
       if (!accumulator->hasValue() ||
-          compareTest(compare(accumulator, decoded, i))) {
+          compareTest(prestosql::compare(accumulator, decoded, i))) {
```

Comment:
perhaps, move the outer class to facebook::velox::aggregate::prestosql namespace

Reply:
Sorry, I do not understand this comment here.
Do you mean to move `NonNumericMinMaxAggregateBase` out of the unnamed namespace and move it to `facebook::velox::aggregate::prestosql` namespace directly?

Reply:
I'm asking why do we need prestosql:: here? I think this shouldn't be needed as Compare and this class should be in the same namespace.

Reply:
Got it thanks. I've removed all the redundant prefixes.